modules/ask_va_api/spec/app/lib/ask_va_api/inquiries/status/retriever_spec.rb

```diff
@@ -6,45 +6,50 @@ module AskVAApi
   module Inquiries
     module Status
       RSpec.describe Retriever do
-        subject(:retriever) { described_class.new(icn: '1') }
+        subject(:retriever) do
```

Comment:
Can you also add a test where icn is not provided/nil?

Reply:
updated